packages/contracts-bedrock/.gas-snapshot

```diff
@@ -158,7 +158,7 @@ L2OutputOracleTest:test_constructor_badTimestamp_reverts() (gas: 70767)
 L2OutputOracleTest:test_constructor_l2BlockTimeZero_reverts() (gas: 45786)
 L2OutputOracleTest:test_constructor_succeeds() (gas: 33695)
 L2OutputOracleTest:test_deleteL2Outputs_afterLatest_reverts() (gas: 211855)
-L2OutputOracleTest:test_deleteL2Outputs_ifNotChallenger_reverts() (gas: 18883)
```

Comment:
LOL, I'm so curious what changed. 
Almost tempted to check-in the `ir` output so we can see the results of such things as they happen.